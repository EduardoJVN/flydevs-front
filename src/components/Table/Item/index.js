import react , {Fragment} from 'react'
import {Button,Card, TextField, Container, Box, CircularProgress , Grid} from '@material-ui/core';
import Typography from '@material-ui/core/Typography';
export default function Item ({pokemons}){
    return(
        <Fragment>
            {pokemons.map((data, index) => (
                <Grid item xs={12} className="itemList" key={index}>
                    <Box border={1}>
                        <Grid container>
                            <Grid item xs={4} sm={4} md={2} lg={2} xl={2}>
                                <img style={{ width: '100%', height: 150 }}  src={data.image} className="image" />
                            </Grid>
                            <Grid item xs={8} sm={8} md={10} lg={10} xl={10}>
                                <Typography component="h5" variant="h5">
                                    {data.name}
                                </Typography>
                            </Grid>
                        </Grid>
                    </Box>
                </Grid>
            ))}
        </Fragment>
        
    )
}