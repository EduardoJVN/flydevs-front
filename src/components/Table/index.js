import react from 'react'
import {Button,Card, TextField, Container, Box, CircularProgress , Grid} from '@material-ui/core';
import Typography from '@material-ui/core/Typography';
export const Table = ({quantity}) =>{
    
    return(
        <Grid item xs={12}>
            <Typography variant="h6" component="h1" gutterBottom>
                {quantity > 0 ? 'Resultados de la búsqueda' : 'No hay resultados de la búsqueda'}
                
            </Typography>
        </Grid>
    )
}