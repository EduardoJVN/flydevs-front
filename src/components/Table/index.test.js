import React from 'react';
import Table from './index';
import ReactDOM from 'react-dom';
import Enzyme , {render} from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
Enzyme.configure({ adapter: new Adapter() });

it('renders correctly', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Table />,div);
    expect(div).toMatchSnapshot();
});

// it('renders Table correctly', () => {
//     const value = 1
//     const wrapper = render(<Table quantity={value}/>);
//     if(value){
//         expect(wrapper.find('.tableTitle').text()).toBe('Resultados de la búsqueda')
//     }else{
//         expect(wrapper.find('.tableTitle').text()).toBe('No hay resultados de la búsqueda')
//     }
    
// });