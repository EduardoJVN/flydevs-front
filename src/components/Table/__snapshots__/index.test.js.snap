// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
  >
    <h1
      class="MuiTypography-root tableTitle MuiTypography-h6 MuiTypography-gutterBottom"
    >
      No hay resultados de la b√∫squeda
    </h1>
  </div>
</div>
`;
